gwy/test.sh

```diff
@@ -1,19 +1,19 @@
-#!/bin/bash
-set -e
-set -x
+#!/bin/bash -ex
 
-if [[ $# -ne 1 ]]; then
-  echo "Usage: test.sh container"
-  exit 1
-fi
+CONTAINER=${CONTAINER}
 
+if [ -z "${CONTAINER}" ]; then
+    echo "You must specify a build container with \${CONTAINER} to test (see my README.md)"
+    exit 1
+fi
 
-CONTAINER=$1
 HEADERS_FILE="./.headers"
 SOME_RESP_HEADER="SOME-RESPONSE-HEADER"
 
+pushd "gwy"
+
 # Test building the gateway.
-docker run --rm -v=`pwd`:/defs $CONTAINER -f test/test.proto -i . -s Message
+docker run --rm -v=$(pwd):/defs "$CONTAINER" -f /defs/test/test.proto -i /defs -s Message
```

Comment:
Probably should be double-quoted, in case there's a space or something in the cwd, and `$PWD` works fine (without spawning a shell) AFAIK, so maybe `"$PWD"`?

Reply:
So when I do this locally, that is an empty string in these scripts...

Reply:
Oh wait, is it case sensitive? Let me try that